pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go

```diff
@@ -23,7 +23,9 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/xanzy/go-cloudstack/cloudstack"
+
 	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/cloudprovider"
```

Comment:
This should be 
```go
cloudprovider k8s.io/cloud-provider
```

Reply:
This is a new repo? I can not see any others packages import it.

Reply:
Yeah there's quite of a few now, maybe you need to pull latest master?

Reply:
Done.